.github/pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+# Pull Request Summary
+(Instructions: this, and all subsequent sections of text should be removed and filled in as appropriate.)   
+Please describe the PR summary
+
+## Description
+Provide a detailed description of what this PR does.
+What bug does it fix, or what feature does it add?
+Is a change of answers expected from this PR?
+
+### Issue(s) addressed
+* Is there an issue associated with this development (bug fix, enhancement, new feature)?    
+Please add a reference to a related issue(s) in WW3 repository (Follow [link](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue)).
+Link the issues to be closed with this PR, whether in this repository, or in another repository.
+(Remember, issues should always be created before starting work on a PR branch!).  
+Note that properly "linked issues" (either automatic links, or manual ones using the correct keywords) will be automatically closed when the PR is merged.
+
+- fixes #<issue_number>
+- fixes noaa-emc/ww3/issues/<issue_number>
+
+### Check list  
+* Is your feature branch up to date with the authoritative repository (NOAA/develop)?
+* Make sure you have checked the [checklist for a developer submitting to develop](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-a-developer-submitting-to-develop), [checklist for a developer submitting to develop](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-a-developer-submitting-to-develop) and [updating version number](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-updating-version-number)
```

Comment:
Maybe we should add the checklist here? Or would that then make the checklist be in two places? 

Also is there a way to preview what this will actually look like?

Reply:
On the top right, click on ... and select _view file_
You will see the final form.
BTW, I was thinking of having the check list here, but then we need to maintain two places, and then whoever does the PR should delete lines and leave the answers. So, if you still like that way, I can do it.

Reply:
I hear you on not wanting to maintain that in 2 places. So sounds good and thanks for showing me how to view the interface!

Reply:
Great, could you approve the review then?